packages/ndla-ui/src/Resource/storyComponents.tsx

```diff
@@ -22,6 +22,7 @@ export const resourceTypesArr = [
   { id: "urn:resourcetype:reviewResource", name: "Vurderingsressurs" },
   { id: "urn:resourcetype:externalResource", name: "Ekstern læringsressurs" },
   { id: "urn:resourcetype:SourceMaterial", name: "Kildemateriale" },
+  { id: "folder", name: "folder" },
```

Comment:
Tipper denne endringa ikkje trengs. Dei andre felta her er taksonomiting og vi har ikkje oppdatert denne for andre favorittmerkinger.

Reply:
Dette er bare for å visualisere den i Storybook:
<img width="1434" alt="image" src="https://github.com/NDLANO/frontend-packages/assets/33912044/6241ceef-6855-4a66-ba4e-50afadb90720">

Eller mente du at jeg sku gjøre det på en annen måte?

Reply:
Ok. Men då kunne vi likegjerne hatt tilsvarende for alle typene i contenttypebadge.